README.md

```diff
@@ -1,4 +1,16 @@
-# az-icons
-Can be used with Arizona Digital CDN link.
+# Arizona Icons
+
+## Using the Icons
+The icons can be used with Arizona Digital CDN link:
 
 https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css
+
+
+## Icon Creation Process
+The icons were designed using vector design software, and the font files were generated using the [IcoMoon App](https://icomoon.io/app).
+
+Before generating the font, the imported files must be properly processed first as *compound path* SVG files on the same sized, square artbord. Strokes in the SVG files will be ignored when IcoMoon generates the files and must be converted to fills and/or compound paths before importing. Once each icon is saved as its own SVG file, they can be imported to IcoMoon and mapped to Unicode's Private Use Area (PUA) code points. This means the glyphs will not be associated with any letters or characters which may be otherwise used symantically.
```

Comment:
```suggestion
Before generating the font, the imported files must be properly processed first as *compound path* SVG files on the same sized, square artboard. Strokes in the SVG files will be ignored when IcoMoon generates the files and must be converted to fills and/or compound paths before importing. Once each icon is saved as its own SVG file, they can be imported to IcoMoon and mapped to Unicode's Private Use Area (PUA) code points. This means the glyphs will not be associated with any letters or characters which may be otherwise used symantically.
```